lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -133,29 +133,30 @@ def expects_uri?
 
         def extract_uri
           return @uri = RDF::URI.new(id) if expects_uri?
-          extract_uri_by_id
+          term_config.term_results_id_predicates.each do |id_predicate|
+            extract_uri_by_id(id_predicate)
+            break if @uri.present?
+          end
+          raise Qa::DataNormalizationError, "Unable to extract URI based on ID: #{id}" if @uri.blank?
+          @uri
         end
 
-        def extract_uri_by_id
-          @uri = get_uri_from_graph_by_id(id)
-        rescue Qa::DataNormalizationError
-          raise unless loc?
-          @uri = get_uri_from_graph_by_id(loc_id)
+        def extract_uri_by_id(id_predicate)
+          @uri = graph_service.subjects_for_object_value(graph: @filtered_graph,
+                                                         predicate: id_predicate,
+                                                         object_value: URI.unescape(id)).first
+          return if @uri.present? || !loc?
+          @uri = graph_service.subjects_for_object_value(graph: @filtered_graph,
+                                                         predicate: id_predicate,
+                                                         object_value: URI.unescape(loc_id)).first
+          return if @uri.blank? # only show the depercation warning if the loc_id was used
           Qa.deprecation_warning(
             in_msg: 'Qa::Authorities::LinkedData::FindTerm',
             msg: 'Special processing of LOC ids is deprecated; id should be an exact match of the id in the graph'
           )
           @uri
         end
 
-        def get_uri_from_graph_by_id(gid)
```

Comment:
With the multiple predicate processing moving the exception up to extract_uri, this method became a single line.  So it was collapsed back into extract_uri_by_id.